vdr/doc/resolvers.go

```diff
@@ -40,6 +40,53 @@ func (d Resolver) Resolve(id did.DID, metadata *types.ResolveMetadata) (*did.Doc
 	return d.Store.Resolve(id, metadata)
 }
 
+// ResolveControllers accepts a list of documents and finds their controllers
```

Comment:
godoc still incorrect (list of documents)

Reply:
removed it, since the doc is correct on the interface